lib/screens/profile_screen.dart

```diff
@@ -197,23 +198,37 @@ class ProfileScreen extends StatelessWidget {
   }
 
   Future<void> signInWithApple() async {
-    final rawNonce = generateNonce();
-    final nonce = sha256ofString(rawNonce);
-
-    final appleCredential = await SignInWithApple.getAppleIDCredential(
-      scopes: [
-        AppleIDAuthorizationScopes.email,
-        AppleIDAuthorizationScopes.fullName,
-      ],
-      nonce: nonce,
-    );
-    // Create an `OAuthCredential` from the credential returned by Apple.
-    final oauthCredential = OAuthProvider('apple.com').credential(
-      idToken: appleCredential.identityToken,
-      rawNonce: rawNonce,
-    );
+    // Sign in the user with Firebase.
+    if (kIsWeb) {
+      final provider = OAuthProvider("apple.com")
```

Comment:
`dart analyze` is reporting a [`prefer_single_quotes`](https://dart-lang.github.io/linter/lints/prefer_single_quotes.html) warning here. We are using single quotes everywhere else.